docs/guides/data-sync/version-control-guide.mdx

```diff
@@ -10,72 +10,74 @@ import { ProOnly } from '/snippets/pro/only.mdx';
+    src="https://mage-ai.github.io/assets/pro/features/deployments/setup-connection-091925.png"
   />
 </Frame>
 
-**Connect to Github using Oauth tokens:**
-
 Once you are in the connect repository UI take the following steps:
 
-**Step 1**: Set your directory as the user code path you intend to use for your project. 
+**Step 1**: Set your directory as the user code path you intend to use for your project.
 
-- In Mage Pro the default directory will be `/home/src/default_repo`.
-- If you changed the `USER_CODE_PATH` environment variable in the Mage Cloud Management Portal, your directory will need to reflect that change.
+- In Mage Pro the default directory will be **`/home/src/default_repo`**.
+- If you changed the **`USER_CODE_PATH`** environment variable in the Mage Cloud Management Portal, your directory will need to reflect that change.
+
+**Step 2 (optional)**: Set your target directory for your deployment if it will be different from your top level directory.
+
+- If you have a nested GitHub project, you may want to set the target directory to `/home/src`
+
+
+<Frame>
+  <img
+    alt="Mage Pro deployments button"
+    src="https://mage-ai.github.io/assets/pro/features/deployments/connect-repo-091925.png"
+  />
+</Frame>
+
+
+## 1.1 Syncing to Github
 
-**Step 2**: Click the “Connect” button to the right of the GitHub connection 
+**Step 1**: Click the “Connect” button to the right of the GitHub connection 
 
 - This will take you to the GitHub sign-in authentication
 
-**Step 3**: Sign in with your Github login credentials
+**Step 2**: Sign in with your Github login credentials
 
 - After authenticating you will return to the Mage-Pro deployment app
 - You should now see that you are connected to Github
 
-**Step 4**: Select the your Github organization from the dropdown list. 
+**Step 3**: Select the your Github organization from the dropdown list. 
 
 - If there are no organizations listed, add an organization to the text field below and click the Set organization button
 
-<Frame>
-  <img
-    alt="Mage Pro connect repo page"
-    src="https://mage-ai.github.io/assets/pro/features/deployments/connect-repo-page.png"
-  />
-</Frame>
-
-**Step 5**: Select the repo you want to commit code to by clicking the repository drop down box
+**Step 4**: Select the repo you want to commit code to by clicking the repository drop down box
 
 - You should see all your Github repos here if you authenticated correctly
 
-**Step 6**: Select your branch (usually main or master) to commit your code to by clicking the dropdown box and selecting the branch name
+**Step 5**: Select your branch (usually main or master) to commit your code to by clicking the dropdown box and selecting the branch name
 
-**Step 7**: Click “Next step to deployment stage” button to move onto the next stage of the workflow.
+**Step 6**: Click “Next step to deployment stage” button to move onto the next stage of the workflow.
```

Comment:
@colefreeman Need to update this step. We no longer need deployment stages, so after selecting a branch, users can go straight to Deployments.
<img width="168" height="56" alt="image" src="https://github.com/user-attachments/assets/a4a37733-86ad-4230-ad52-6a99097636a2" />